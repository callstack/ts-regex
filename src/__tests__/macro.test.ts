import pluginTester from 'babel-plugin-tester';
import plugin from 'babel-plugin-macros';

pluginTester({
  plugin,
  snapshot: true,
  babelOptions: { filename: __filename },
  tests: [
    `
      import MyMacro from '../macro.cts'

      MyMacro({someOption: true}, \`
        some stuff
      \`)
    `,
  ],
});
