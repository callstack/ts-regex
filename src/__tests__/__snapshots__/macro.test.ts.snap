// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `


import MyMacro from '../macro.cts'

MyMacro({someOption: true}, \`
  some stuff
\`)
    

      ↓ ↓ ↓ ↓ ↓ ↓

import MyMacro from '../macro.cts';
MyMacro(
  {
    someOption: true,
  },
  \`
  some stuff
\`,
);

`;
